share/docker/Dockerfile

```
@@ -69,7 +69,7 @@ RUN apt-get -qqy update \
  && rm -rf /var/lib/apt/lists
 
 # Install nodejs 6
-RUN curl -sSL https://deb.nodesource.com/setup_6.x \
+RUN curl -sSL https://deb.nodesource.com/node_10.x \
```

```suggestion
RUN curl -sSL https://deb.nodesource.com/setup_10.x \
```
I think you meant this URL!  Comment to be updated...

---

thanks, done